webapp/src/component/layout/Notifications/NotificationItem.tsx

```diff
@@ -75,34 +87,72 @@ export const NotificationItem: React.FC<NotificationItemProps> = ({
       data-cy="notifications-list-item"
     >
       <StyledAvatar>
-        {originatingUser && (
-          <AvatarImg
-            owner={{
-              name: originatingUser.name,
-              avatar: originatingUser.avatar,
-              type: 'USER',
-              id: originatingUser.id || 0,
-            }}
-            size={32}
+        {notification ? (
+          originatingUser && (
+            <AvatarImg
+              owner={{
+                name: originatingUser.name,
+                avatar: originatingUser.avatar,
+                type: 'USER',
+                id: originatingUser.id || 0,
+              }}
+              size={32}
+            />
+          )
+        ) : (
+          <LoadingSkeleton
```

Comment:
Can we move the notification skeleton to separate component so it doesn't mix with the actual code?